package.json

```diff
@@ -1488,6 +1488,11 @@
           "when": "view == projectExplorer && viewItem =~ /^objectFile.FILE.SPF.*/",
           "group": "4_search@0"
         },
+        {
+          "command": "vscode-ibmi-projectexplorer.download",
+          "when": "view == projectExplorer && viewItem =~ /^objectFile.FILE.SPF.*/ && !vscode-ibmi-projectexplorer:isInMerlin",
+          "group": "5_transfer@0"
+        },
```

Comment:
so now we can download source members to the workspace?

Reply:
We already had this download action for members, but forgot to contribute the same action on source physicals (this PR fixes that).

However, this `Download` action as well as the `Upload and Replace` action seem to have been disabled when we are in Merlin in these PRs -> https://github.com/IBM/vscode-ibmi-projectexplorer/pull/420, https://github.com/IBM/vscode-ibmi-projectexplorer/pull/427

@edmundreinhardt In an old issue, you mentioned: "The current action prompts for files from the Merlin container rather that workstation and so is not useful in Merlin" (https://github.com/IBM/vscode-ibmi-projectexplorer/issues/396#issuecomment-2018602538). I just tested both the download and upload now in Merlin using the Object Browser and we can use the files in the workspace, but the default path (for the upload scenario) is not the current workspace which can be inconvenient :

![image](https://github.com/IBM/vscode-ibmi-projectexplorer/assets/32170854/1c5688c9-dbb8-45f8-b171-8e7d7fb8e2ba)

However, the user can still navigate to the current workspace so I think we should add these actions back. What do you think?

I can also see if Code4i's command for upload can be modified to pass in the workspace path.